docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,506 @@
+
+2.  In the configuration editor, view and modify the configuration details using the various tabs:
+
+    ![Configuring the addon](media/basic_addon_config_w.png)
+    > [!NOTE]
+    > The **config.xml** file must conform to both the XML file format and the W3C specification requirements. Editing the file in the **Source** tab is intended for advanced users only.  
+    > You can see an error in preceding image of step 2. However, it does not affect on this process because **Content** is not used for addons.
+
+    -   **Overview**: Define general information, such as the name and icon of the addon.
+
+    -   **Features**: Define required software and hardware features. This information is used for addon filtering in the Tizen Store.
+
+    -   **Privileges**: Define the security-sensitive APIs or API groups accessed and used by the addon.
+
+    -   **Localization**: Define localized values for the addon name, description, and license.
+
+    -   **Policy**: Request network resource permissions to access external network resources.
+
+    -   **Preferences**: Define name-value pairs that can be set or retrieved through the addon.
+
+    -   **Tizen**: Edit the Tizen schema extension properties of the addon.
+
+    -   **Source**: View and edit the source code of the **config.xml** file. Changes made and saved on the other tabs are reflected in the source code and vice versa.
+
+3.  To save any changes, in the Tizen Studio menu, select **File \> Save All**.
+
+For more information on configuring the addon, see [Setting the Web Application Configuration](../../tutorials/process/setting-properties.md#set_widget).
+
+<a name="build"></a>
+## Build your addon package
+
+After you have created the addon project, you can implement the required features. In this example, only the default features from the project template are used, and no code changes are required.
+
+When your addon code is ready, you must build the addon package. The building process performs a validation check and compiles your JavaScript and CSS files.
+
+To build a signed package, right-click the project in the **Project Explorer** view and select **Build Signed Project**.
+
+**Figure: Building the addon**
+
+![Building the addon](media/build_manual_a.png)
+
+After you have built the addon, run it.
+
+<a name="run"></a>
+## Run your addon
+
+You can run the addon on the [emulator](../../tutorials/process/run-debug-app.md#emulator) or a [real target device](../../tutorials/process/run-debug-app.md#target).
+
+<a name="emulator"></a>
+### Run on emulator
+
+To run the addon on the emulator:
+
+1.  Launch an emulator instance in the [Emulator Manager](../../../tizen-studio/common-tools/emulator-manager.md):
+    1. In the Tizen Studio menu, select **Tools \> Emulator Manager**.
+
+        ![Emulator Manager](media/emulator_icon.png)
+
+    2.  In the Emulator Manager, select a mobile emulator from the list and click **Launch**.
```

Comment:
Please remove "the" before "Emulator Manager".

Reply:
done including the same in line number 154